doc/source/install.rst

```diff
@@ -47,13 +47,43 @@ Execute the following command from the shell::
 
 Windows
 ```````
+Before installing the development version, uninstall the standard version of
```

Comment:
Aah. Right, because on virtual environment you can just have the packages you need. I think it does not pre-install any packages. 

Yes, the update process should be added as well. Thank you.